src/staticanalysis/bot/static_analysis_bot/coverity/coverity.py

```diff
@@ -351,8 +357,12 @@ def as_phabricator_lint(self):
         '''
         Outputs a Phabricator lint result
         '''
+        # If there is the reliability index use it
+        message = 'Checker reliability (false positive risk) is {}.'.format(
+            self.reliability.value) + self.reliability if self.reliability != Reliability.Unknown else self.message
```

Comment:
Nit: that's pretty hard to read, could you put in on several lines so it's easier to parse by humans ?